Python/Product/Analysis/Values/BuiltinModule.cs

```diff
@@ -33,9 +36,23 @@ public BuiltinModule(IPythonModule module, PythonAnalyzer projectState)
 
         public IPythonModule InterpreterModule => _interpreterModule;
 
+        public void AddChildModule(string memberName, AnalysisValue module) {
+            if (_childModules == null) {
+                _childModules = new Dictionary<string, IAnalysisSet>();
+            }
+            if (_childModules.TryGetValue(memberName, out var existing)) {
```

Comment:
Sequence is getting popular... Might we worth writing extension to dictionary like `Add<T>(T value, Func<T, T> newValueConstructor);` where the new value constructor is called to make the actual value when existing is there.